x-pack/dockerlogbeat/config.json

```diff
@@ -0,0 +1,24 @@
+{
+    "description": "A little example log driver for docker",
+    "documentation": "https://docs.docker.com/engine/extend/plugin_api/",
```

Comment:
we will need to update this info at some point, no need to do it in this PR but please add a note somewhere